RedHat-UX/red-hat-design-system
# docs(pagination): additional, minor fixes

## What I did

1. Fixed inadvertent indentation of image following a bulleted list
2. Replaced sections of images with a single image instead
3. Deleted aforementioned images

## Testing Instructions

1. Verify the [changes](https://github.com/RedHat-UX/red-hat-design-system/pull/993#issuecomment-1592430977) requested by Corey
